libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -650,7 +650,7 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 			{
 				// Check if zero (out of stack or not enough balance).
 				m_context << Instruction::ISZERO;
-				m_context.appendConditionalInvalid();
+				m_context.appendConditionalRevert();
```

Comment:
@chriseth I'm also not completely sure about this case. Is the reasoning here similar to the one for contract creations?